pgd/settings.py

```diff
@@ -128,6 +128,7 @@
     'django.middleware.common.CommonMiddleware',
     'django.contrib.sessions.middleware.SessionMiddleware',
     'django.contrib.auth.middleware.AuthenticationMiddleware',
+    'django.contrib.messages.middleware.MessageMiddleware',
```

Comment:
Added this because I got an error when I tried adding users from Admin(for testing the features)